prototypes/pipe-clamps.lua

```diff
@@ -161,6 +161,42 @@ local function build_picture_table(current_picture_table, current_name)
     }
 end
 
+local function build_composite_icon(current_entity)
+    local new_icons = {}
+
+    if current_entity.icons then
+        -- Icons table, transcribe layers into new_icons, fully define the IconData
+        for _, icon_data in pairs(current_entity.icons) do
+            table.insert(new_icons, {
```

Comment:
Replace table.insert with t[#t + 1] =

Reply:
I can make this change, though I'd like to understand why? The `new_icons` table is initialize with no entries, so the inserted table will always be at the first position.

Reply:
More of an aesthetics opinion issue. I.e I like the way it looks.

Also t[#t +1] is slightly almost unnoticeable faster and even slightly unnoticeable faster if #t is localed and incrememnted inside the funciton (not that it matters in this case though)